org.zaproxy.ZAP.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "org.zaproxy.ZAP",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "zap",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11" 
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--env=JAVA_HOME=/app/jre",
+        "--filesystem=home"
```

Comment:
Does it need access to entire home or could be limited?

Reply:
Great question, you are right to point this out. There does need to be some interaction with the home directory (to save sessions, upload target lists, etc) but it doesn't need much more that Downloads and Documents for those purposes. I've amended the manifest.

Reply:
ZAP creates a directory in the user's home (for main releases called `.ZAP`, which has the add-ons and other configuration files) would that still work with the new settings?

Reply:
After `home` was replaced with `xdg-document` and `xdg-download`, no. You need to add `--persist=.ZAP` and flatpak will handle binding appropriate location.

In general it's good practice to limit filesystem access. Users can always run `flatpak override --filesystem=home org.zaproxy.ZAP` (or `host`) on their own.